Update pages (#42)
Vanilla-OS/handbook: _posts/2023-01-07-install-steam.md

## What is Steam?

- [**Steam**](https://steampowered.com) is the most popular gaming platform. It allows you to download and play a vast library of games. It's client is available natively on Linux.

> - On 21 August 2018, Steam introduced the [Proton](https://github.com/ValveSoftware/Proton) project, which allows you to run Windows games on Linux. It is an essential feature for Linux gamers as it extends the number of games available on the platform. Check out [**ProtonDB**](https://www.protondb.com) to see how well your favourite games work with Proton.

Reviewer: ```suggestion
- On 21 August 2018, Steam introduced the [Proton](https://github.com/ValveSoftware/Proton) project, which allows you to run Windows games on Linux. It is an essential tool for Linux gamers as it extends the number of games available on the platform. Check out [**ProtonDB**](https://www.protondb.com) to see how well your favourite games work with Proton.
```

## Install Steam

> In Vanilla OS there are many ways to install applications. The easiest way is to use

Reviewer: ```suggestion
In Vanilla OS there are many ways to install Steam. The easiest way is to use
```

[Flatpak](https://flatpak.org), which is a universal package manager for Linux.
But it is also possible to install Steam using [Apx](https://documentation.vanillaos.org/docs/apx/)
the default package manager of Vanilla OS.

### Flatpak

If you chose to install Flatpak during [**First Setup**](/2022/11/18/first-setup.html), 
you will be able to install applications directly from GNOME Software, as 
explained [**here**](/2022/12/09/install-flatpaks.html#title3).

You can also install the Steam Flatpak from the terminal using the following command:

```bash
flatpak install flathub com.valvesoftware.Steam
```

### Apx

> **Apx** is a package manager which installs software in containers thanks to

Reviewer: ```suggestion
**apx** is a package manager which installs software in containers thanks to
```

> [Distrobox](https://github.com/89luca89/distrobox). In our tests, Fedora is the best distro to run Steam in a container . We highly recommend using it to install Steam.

Reviewer: ```suggestion
[Distrobox](https://github.com/89luca89/distrobox). In our tests, Fedora is the best distro to run Steam in a container. We highly recommend using it to install Steam.
```

apx --dnf xorg-x11-drv-intel mesa-libGL.i686 mesa-dri-drivers.i686
```

> - Now we can install Steam using the following command:

Reviewer: ```suggestion
- We can now install Steam using the following command:
```

```bash
apx --dnf install steam
```

##### Run Steam

- To run Steam, you can use the icon from the applications menu or issue the 
command `apx --dnf run steam`.

- If the icon doesn't appear in the applications menu, you can add it manually using the following command:

```bash
apx --dnf export steam
```

#### Ubuntu container

> - On Ubuntu, we need to add the `multiverse` repository to our sources list using the follow:

Reviewer: ```suggestion
- On Ubuntu, we need to add the `multiverse` repository to our sources list using the following command:
```

apx install nvidia-driver-525
```

- **AMD/Intel**:-

```bash
apx --apt install mesa-utils mesa-utils-extra
```

> Now we can install Steam using the following command:

Reviewer: ```suggestion
- We can now install Steam using the following command:
```
